t/macros/load_macros.t

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env perl
+
+=head1 Test macro loading
+
+This tests the pg/macro files to see that they all load without errors or
+warnings.
+
+Note that macros that are broken are listed in C<%brokenMacros> and testing of
+these macros is skipped.  These macros should be fixed or considered for
+deletion.
+
+=cut
+
+die "PG_ROOT not found in environment.\n" unless $ENV{PG_ROOT};
+use lib "$ENV{PG_ROOT}/lib";
+
+use Test2::V0;
+use File::Find;
+use WeBWorK::PG;
+
+my %baseMacros = ('PG.pl' => 1, 'PGstandard.pl' => 1);
+
+my %brokenMacros = ('answerDiscussion.pl' => 1);
+
+die "Unable to open pg macro directory: $ENV{PG_ROOT}/macros" unless -e "$ENV{PG_ROOT}/macros";
+
+# Find all macro files inside the $ENV{PG_ROOT}/macros directory.
+my @macro_files;
+find ( sub {
+		my $f = $_;
```

Comment:
This line is not needed with the other changes suggested here.

Reply:
I noticed this. Will push changes soon.